metadata.py

```diff
@@ -0,0 +1,64 @@
+"""
+The purpose of this file is to define the metadata of the app with minimal imports. 
+
+DO NOT CHANGE the name of the file
+"""
+
+from mmif import DocumentTypes, AnnotationTypes
+
+from clams.app import ClamsApp
+from clams.appmetadata import AppMetadata
+from lapps.discriminators import Uri
+
+
+# DO NOT CHANGE the function name 
+def appmetadata() -> AppMetadata:
+    """
+    Function to set app-metadata values and return it as an ``AppMetadata`` obj.
+    Read these documentations before changing the code below
+    - https://sdk.clams.ai/appmetadata.html metadata specification. 
+    - https://sdk.clams.ai/autodoc/clams.appmetadata.html python API
+    
+    :return: AppMetadata object holding all necessary information.
+    """
+    
+    # first set up some basic information
+    metadata = AppMetadata(
+        name="Tfidf Keyword Detector",
+        description="",  # briefly describe what the purpose and features of the app
```

Comment:
say something about the app, maybe including some "documentation" of the training data and model status, etc.